Keas.Mvc/Controllers/Api/DocumentController.cs

```diff
@@ -10,14 +10,14 @@
 using System.Threading.Tasks;
 using Microsoft.EntityFrameworkCore;
 using System;
+using System.Net.Mime;
 
 namespace Keas.Mvc.Controllers.Api
 {
     [Authorize(Policy = AccessCodes.Codes.DocumentMasterAccess)]
     [ApiController]
     [Route("api/{teamName}/documents/[action]")]
-    [Consumes("application/json")]
-    [Produces("application/json")]
+    [Produces(MediaTypeNames.Application.Json)]
     public class DocumentsController : SuperController
```

Comment:
I'm not sure it's good to have Produces at the controller level for this controller. The Get action shows it produces both application/pdf and application/json. But I don't see how it can produce json.

Reply:
@sprucely It was Scott's suggestion.